lib/src/hydrated.dart

```diff
@@ -58,21 +47,23 @@ typedef HydrateCallback<T> = T Function(String);
 ///   );
 /// ```
 class HydratedSubject<T> extends Subject<T> implements ValueStream<T> {
-  static final _areTypesEqual = TypeUtils.areTypesEqual;
-  final BehaviorSubject<T> _subject;
   final String _key;
   final HydrateCallback<T>? _hydrate;
   final PersistCallback<T>? _persist;
+  final BehaviorSubject<T> _subject;
   final VoidCallback? _onHydrate;
   final T? _seedValue;
 
```

Comment:
```suggestion
```

Reply:
Not sure this is possible -- we are delegating several methods to `_subject` in order to conform to the `Subject<T>` interface.

Unfortunately, it is not possible to extend `BehaviorSubject` directly, so we are forced to create tramp methods, such as this:

```dart
  @override
  bool get hasValue => _subject.hasValue;
```